WORKSPACE

```diff
@@ -10,8 +10,7 @@ http_archive(
 
```

Comment:
Please add comment here saying we cannot update until https://github.com/bazelbuild/bazel-gazelle/issues/272 is resolved.